cmd/uhc/cluster/list/list.go

```diff
@@ -130,7 +130,11 @@ func run(cmd *cobra.Command, argv []string) {
 	colUpper = strings.Replace(colUpper, ".", " ", -1)
 	columnNames := strings.Split(colUpper, ",")
 	paddingByColumn := []int{35, 25, 70, 25, 15}
-	if args.columns != "id,name,api.url,version.id,region.id" {
+	if args.padding != -1 {
+		if args.padding < 2 {
+			fmt.Printf("padding flag needs to be an integer greater than 2\n")
```

Comment:
As this is a message displayed to the user directly it should start with upper case.